concrete-core-fixture/src/fixture/lwe_ciphertext_vector_discarding_affine_transformation.rs

```diff
@@ -0,0 +1,222 @@
+    Fixture<
+        Precision,
+        Engine,
+        (
+            CiphertextVector,
+            CleartextVector,
+            Plaintext,
+            OutputCiphertext,
+        ),
+    > for LweCiphertextVectorDiscardingAffineTransformationFixture
+where
+    Precision: IntegerPrecision,
+    Engine: LweCiphertextVectorDiscardingAffineTransformationEngine<
+        CiphertextVector,
+        CleartextVector,
+        Plaintext,
+        OutputCiphertext,
+    >,
+    CiphertextVector: LweCiphertextVectorEntity,
+    CleartextVector: CleartextVectorEntity,
+    Plaintext: PlaintextEntity,
+    OutputCiphertext: LweCiphertextEntity<KeyDistribution = CiphertextVector::KeyDistribution>,
+    Maker: SynthesizesLweCiphertextVector<Precision, CiphertextVector>
+        + SynthesizesCleartextVector<Precision, CleartextVector>
+        + SynthesizesPlaintext<Precision, Plaintext>
+        + SynthesizesLweCiphertext<Precision, OutputCiphertext>,
+{
+    type Parameters = LweCiphertextVectorDiscardingAffineTransformationParameters;
+    type RepetitionPrototypes = (
+        <Maker as PrototypesLweSecretKey<Precision, CiphertextVector::KeyDistribution>>::LweSecretKeyProto,
+        <Maker as PrototypesCleartextVector<Precision>>::CleartextVectorProto,
+        <Maker as PrototypesPlaintext<Precision>>::PlaintextProto
+    );
+    type SamplePrototypes = (
+        <Maker as PrototypesLweCiphertext<Precision, CiphertextVector::KeyDistribution>>::LweCiphertextProto,
+        <Maker as PrototypesPlaintextVector<Precision>>::PlaintextVectorProto,
+        <Maker as PrototypesLweCiphertextVector<Precision, CiphertextVector::KeyDistribution>>::LweCiphertextVectorProto,
+    );
+    type PreExecutionContext = (
+        OutputCiphertext,
+        CiphertextVector,
+        CleartextVector,
+        Plaintext,
+    );
+    type PostExecutionContext = (
+        OutputCiphertext,
+        CiphertextVector,
+        CleartextVector,
+        Plaintext,
+    );
+    type Criteria = (Variance,);
+    type Outcome = (Precision::Raw, Precision::Raw);
+
+    fn generate_parameters_iterator() -> Box<dyn Iterator<Item = Self::Parameters>> {
+        Box::new(
+            vec![
+                LweCiphertextVectorDiscardingAffineTransformationParameters {
+                    nb_ct: LweCiphertextCount(100),
+                    noise: Variance(LogStandardDev::from_log_standard_dev(-25.).get_variance()),
+                    lwe_dimension: LweDimension(1000),
+                },
+            ]
+            .into_iter(),
+        )
+    }
+
+    fn generate_random_repetition_prototypes(
+        parameters: &Self::Parameters,
+        maker: &mut Maker,
+    ) -> Self::RepetitionPrototypes {
+        let proto_secret_key = maker.new_lwe_secret_key(parameters.lwe_dimension);
+        let raw_cleartext_vector =
+            Precision::Raw::uniform_zero_centered_vec(512, parameters.nb_ct.0);
```

Comment:
The original test seems to have used values between -256 and 256 instead of 512.